src/facade/dragonfly_listener.cc

```diff
@@ -27,6 +26,7 @@ ABSL_FLAG(string, tls_cert_file, "", "cert file for tls connections");
 ABSL_FLAG(string, tls_key_file, "", "key file for tls connections");
 ABSL_FLAG(string, tls_ca_cert_file, "", "ca signed certificate to validate tls connections");
 ABSL_FLAG(string, tls_ca_cert_dir, "", "ca signed certificates directory");
+ABSL_FLAG(uint32_t, tcp_keepalive, 300, "the period in seconds used to send ACKs to client");
```

Comment:
I think the description of this flag is not accurate. 
1. We start sending acks after the idle time that is bound to the value of this flag.,
2. We give up on the connection if we have not got any ack response during the additional period that equals to the same value.

So it's worth to mention that it's a period of inactivity after which keep-alives are triggerred and the total time until the dead connection is closed is twice the specified value.

![image](https://github.com/dragonflydb/dragonfly/assets/3674760/b59c615d-4e3a-45f1-a968-bcac36e0b508)